### DISCUSSION TITLE
nuejs on codepen examples?? [request]

### DISCUSSION THREAD

**Discussion by diomed** [nuejs/nue#53] [Q&A]
hello

could anyone do a few of examples of this library on codepen?

**Comment by tipiirai** [comment]
Eventually I hope the community starts producing this. I'm personally focusing on the website. Many examples already here: https://nuejs.org/docs/nuejs/examples/

**Comment by diomed** [comment]
my point is - people should see this working in environment that's not official webpage.

**Reply by tipiirai** [reply]
Very good point. Hope someone steps in!
